server/tinode.conf

```diff
@@ -190,7 +190,14 @@
 				// See https://github.com/go-sql-driver/mysql#dsn-data-source-name for syntax.
 				"dsn": "root@tcp(localhost)/tinode?parseTime=true&collation=utf8mb4_unicode_ci",
 				// Name of the main database.
-				"database": "tinode"
+				"database": "tinode",
```

Comment:
Ditto here: the adapter returns an error when `"database"` is specified.

Reply:
Done.